packages/text/src/TextMetrics.js

```diff
@@ -49,6 +49,13 @@ export default class TextMetrics
         wordWrap = (wordWrap === undefined || wordWrap === null) ? style.wordWrap : wordWrap;
         const font = style.toFontString();
         const fontProperties = TextMetrics.measureFont(font);
+
+        if (fontProperties.fontSize === 0)
```

Comment:
Great explanation in your PR description. Could you add a comment inline about why you're doing that here? I think it's very important for context whenever there are weird edge-cases like this.